packages/flutter_tools/lib/src/base/doctor.dart

```diff
@@ -0,0 +1,154 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../android/android_workflow.dart';
+import '../ios/ios_workflow.dart';
```

Comment:
base seems like an odd place for this code.  Maybe just up one level in src?  Dunno.